packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -433,39 +123,23 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
    * calls are cleaned up.
    */
   private callsClearedHandler = async () => {
-    if (!this.registration.isDeviceRegistered()) {
+    // this is a temporary logic to get registration obj
+    // it will change once we have proper lineId and multiple lines as well
+    const {registration} = Object.values(this.lineDict)[0];
```

Comment:
If we do have multiple lines, what is the logic in that case? Why can't we use it now itself?

Reply:
As discussed with Priya, no need to refactor it as anyways this logic is going to change.